src/commands/readme.js

```diff
@@ -94,22 +95,48 @@ module.exports.handler = function readme(argv) {
         .process(readmeContent)
     )
     .then(file => {
-      const diffOutput = disparity.unified(readmeContent, file.contents, {
-        paths: [argv.readmeFile, argv.readmeFile]
-      });
-      if (!diffOutput.length) {
+      const diffRaw = diff.createPatch(
+        '',
+        readmeContent,
+        file.contents,
+        '',
+        ''
+      );
+      if (!diffRaw.length) {
         log(`${argv.readmeFile} is up to date.`);
         process.exit(0);
       }
 
+      // Replace diff headers with real values
+      const cleanedDiff = diffRaw
+        .replace(/^([^\n]+)\n([^\n]+)\n/m, '')
+        .replace(/^---.*/gm, `--- ${argv.readmeFile}\tremoved`)
+        .replace(/^\+\+\+.*/gm, `+++ ${argv.readmeFile}\tadded`);
+
+      // Includes newlines for easier joins
+      const diffLines = cleanedDiff.split(/^/m);
+      const diffHeader = diffLines
+        .slice(0, 2)
+        .join('')
+        .replace(/[^\n\r]+/g, `${ansi.yellow.open}$&${ansi.yellow.close}`);
```

Comment:
Could we use the pre-existing `chalk` dependency for this styling instead of introducing ansi-styles?

Reply:
Good call. My original intention was to keep the logic as close to disparity as possible but using chalk makes much more sense.